### DISCUSSION TITLE
Things we can try from solutions

### DISCUSSION THREAD

**Discussion by mrinal18** [Mrinal18/NFL_Kaggle#21] [Ideas]
@nvk681 @lalalaurenville @gsvsumanth @Archana-bc 

1. Confidence : 0.01 - 0.7 
2. IOU score : 0.01 or 0.35 
3. Angle (havent tried yet)
4. Direction of moment (havent tried yet)
5. Epochs < (10-15)
6. Tracking of helmets 
7. Limited players <= 22 
8. Check for outlier bounding boxes by taking fps as consideration. : needs experimenting. 

**What we have now**
9. Baseline model: 0.6
10. Tracking angles : 0.806

**Something to learn:**
Hungarian Algorithm

**What we can do: Next Wednesday**

1. https://www.kaggle.com/c/nfl-health-and-safety-helmet-assignment/discussion/285286
2. https://www.kaggle.com/c/nfl-health-and-safety-helmet-assignment/discussion/285076
3. https://www.kaggle.com/c/nfl-health-and-safety-helmet-assignment/discussion/285065
4. https://www.kaggle.com/c/nfl-health-and-safety-helmet-assignment/discussion/284940

1. Try with different values IOU, conf 
Tracking angles
 -> Check for outlier bounding boxes by taking fps as consideration. : needs experimenting.  People can split and try to experimenting 
->  Tracking of helmets

Notebooks to work on: 
1. https://www.kaggle.com/mrinalmathur/testing-deepsort/edit/run/77737624 : deep sort + yolo5 only 

2. https://www.kaggle.com/mrinalmathur/deepsort-with-tracking-angles?scriptVersionId=78732225 deepsort + yolo5 + tracking

**Comment by Archana-bc** [comment]
https://docs.google.com/spreadsheets/d/17d-wTenNcnfzKUcUwsmUEKul8qUf4_Pmf9PLEf_cRRk/edit?usp=sharing
use above link to record scores for various iou and conf
